src/test/create-user.test.ts

```diff
@@ -0,0 +1,89 @@
+import { CryptoService } from '@core/crypto';
+import { CRYPTO_KEY_LENGTH, CRYPTO_SECRET } from '@core/env-conf';
+import { UserDataSource } from '@db/source';
+import { CreateUserUseCase } from '@domain/mutation';
+import { expect } from 'chai';
+import Container from 'typedi';
+
+describe('GraphQL -> Create User - use-case', () => {
+  let createUser: CreateUserUseCase;
+  let repository: UserDataSource;
+  let cryptoService: CryptoService;
+
+  beforeEach(() => {
+    const secret = Container.get(CRYPTO_SECRET);
+    const key_length: number = Container.get(CRYPTO_KEY_LENGTH);
+
+    repository = new UserDataSource();
+    cryptoService = new CryptoService(secret, key_length);
+    createUser = new CreateUserUseCase(repository, cryptoService);
+  });
+
+  it('Deve criar um novo usuário', async () => {
```

Comment:
Por padrao, mantemos essas strings em ingles iniciado pela palavra `shuold`:

```typescript
  it('should create a new user', async () => {
    // ...
  });

  it('should throw an error if the email is already in use', async () => {
      // ...
  });
```

lê-se "it should create a new user"